src/app/ApplicationTemplate.Shared.Views/Content/WelcomePage.xaml

```diff
@@ -0,0 +1,30 @@
+﻿<Page x:Class="ApplicationTemplate.Views.Content.WelcomePage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+
+	<Grid Background="{StaticResource AlternateBackgroundBrush}">
```

Comment:
I don't understand why this page is needed. We're not even navigating to it it seems

Reply:
at least yet

Reply:
Weird it's not even showing in my solution.